drivers/wifi/siwx917/Kconfig.siwx917

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 Antmicro
+# SPDX-License-Identifier: Apache-2.0
+
+config WIFI_SIWX917
+	bool "Silabs SiWx917 SoC series WiFi driver"
+	default y
+	depends on DT_HAS_SILABS_SIWX917_WIFI_ENABLED
+	select EVENTS
+	select WIFI_OFFLOAD
+	select NET_L2_WIFI_MGMT
+	select CMSIS_RTOS_V2
+	# Also select CMSIS-RTOS v2 dependencies
+	select POLL
+	select DYNAMIC_THREAD
+	select THREAD_NAME
+	select THREAD_STACK_INFO
+	select THREAD_MONITOR
+	select INIT_STACKS
+	help
+	  Enable WiFi driver for the Silabs SiWx917 SoC series.
+
+if WIFI_SIWX917
+
+# WiseConnect create threads with realtime priority. Default (10kHz) clock tick
```

Comment:
I've looked at implementing the OS timer using sleeptimer backed by sysrtc. This would tick at 1024 Hz, which is the same as FreeRTOS does in Wiseconnect. Is there a reason you needed to reduce it all the way to 100?

Reply:
I wrote that before you fixed the clock issues. I have just run some tests and it seems not true anymore.